ground/src/main/res/layout/question_data_collection_frag.xml

```diff
@@ -21,8 +21,8 @@
 
   <data>
     <variable
-      name="task"
-      type="com.google.android.ground.model.task.Task" />
+      name="viewModel"
+      type="com.google.android.ground.ui.editsubmission.TextTaskViewModel" />
```

Comment:
Would this type eventually be dynamic?

Reply:
I don't think so, since we will have different layouts for the different types of Tasks